cli/azd/pkg/tools/github/github.go

```diff
@@ -311,7 +315,7 @@ func (cli *ghCli) GitHubActionsExists(ctx context.Context, repoSlug string) (boo
 	return true, nil
 }
 
-func (cli *ghCli) ForceConfigureAuth(authMode AuthTokenSource) {
+func (cli *ghCli) forceConfigureAuth(authMode AuthTokenSource) {
```

Comment:
I think with your changes some of this is dead code now.  It looks like we only call this with TokenSourceFile now?  I also think we might be able to just remove the entire thing by moving some of the logic we had which called this into `newRunArgs` as you had done in `git.go`.